lua/xit/init.lua

```diff
@@ -302,9 +294,9 @@ M.setup = function(opts)
   local parser_config = require('nvim-treesitter.parsers').get_parser_configs()
   parser_config.xit = {
     install_info = {
-      url = 'https://github.com/synaptiko/tree-sitter-xit',
+      url = 'https://github.com/felipesere/tree-sitter-xit',
       files = { 'src/parser.c' },
-      revision = '0.1',
+      revision = 'master',
```

Comment:
Would you consider to create PR for `tree-sitter-xit` as well and then we could revert this and I would release `0.2` with your changes?